Client/Frontend/Browser/SavedTab.swift

```diff
@@ -20,32 +23,48 @@ class SavedTab: NSObject, NSCoding {
     var createdAt: Timestamp?
     var hasHomeScreenshot: Bool
 
+    private enum Keys: String {
```

Comment:
Having the class conform to `Codable` should give you these keys under `CodingKeys` instead of having to manually make an enum for each class.

Reply:
@q2r5 This class is using the `NSCoding` protocol instead of the `Codable` protocol, hence not having all the freebies that `Codable` gets. From what I get we could migrate to `Codable` instead of `NSCoding`, but we would still need to define the keys like shown [here](https://www.hackingwithswift.com/example-code/language/how-to-store-nscoding-data-using-codable). If that's the way we wanna go I would do this in another PR.